### PR TITLE
ARXIVNG-3789: tableau membership reporting

### PR DESCRIPTION
**Pull request by afromme**
These changes add 2020 usage data pages and embedded tableau data for downloads, submissions, and institutional data.

### CODE REVIEW

**Review by mhl10 — APPROVED** [review]
Super!

**Review by bmaltzan — APPROVED** [review]
Looks good!

**Review by mhl10 — CHANGES_REQUESTED** [review]
Before we merge this PR I would like to investigate a problem with the Tableau data reported by @ep538 (2020 showing drop in submissions compared to previous year). @bmaltzan I may need your help if a fix is needed.

**Comment by mhl10** [comment]
> Before we merge this PR I would like to investigate a problem with the Tableau data reported by @ep538 (2020 showing drop in submissions compared to previous year). @bmaltzan I may need your help if a fix is needed.

The issue is in the submission by institution data; the 2020 data (from Microsoft) appears to be incomplete.

File: about/reports/2020_institution_submissions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Submissions by Institution in 2020
+
+_See more [arXiv in Numbers](/about/reports/2020_usage)_
+
+The following table shows submissions by institutions in 2018, 2019, and 2020. To search for a specific institutional domain, hover your mouse over the right hand corner of the list of institutions to reveal the search icon.
```
Comment (mhl10) [inline about/reports/2020_institution_submissions.md:5]: @afromme we should indicate somewhere on this page that the data comes from Microsoft Analytics

**Comment by mhl10** [comment]
> > Before we merge this PR I would like to investigate a problem with the Tableau data reported by @ep538 (2020 showing drop in submissions compared to previous year). @bmaltzan I may need your help if a fix is needed.
> 
> The issue is in the submission by institution data; the 2020 data (from Microsoft) appears to be incomplete.

As of today I haven't heard back from Microsoft with an explanation, so I would recommend either excluding this charge or making it clear that the data for 2020 is still incomplete.

**Review by mhl10 — APPROVED** [review]

File: about/reports/2020_institution_submissions.md (hunk 1 above)
Comment (afromme) [reply about/reports/2020_institution_submissions.md]: Yes, agreed and done

**Pull request merged**